pkg/trust/policy.go

```diff
@@ -55,6 +59,10 @@ type genericRepoMap map[string]json.RawMessage
 // DefaultPolicyPath returns a path to the default policy of the system.
 func DefaultPolicyPath(sys *types.SystemContext) string {
 	systemDefaultPolicyPath := config.DefaultSignaturePolicyPath
+	userPolicyFilePath := filepath.Join(homedir.Get(), userPolicyFile)
+	if _, err := os.Stat(userPolicyFilePath); err == nil {
+		return userPolicyFilePath
+	}
 	if sys != nil {
```

Comment:
https://github.com/containers/podman/issues/19073#issuecomment-1632731876 quotes the c/image implementation. I think this should be prioritizing alternatives the same way, and it isn’t doing so.

Reply:
Yup, totally my bad, had a dirty stash and applied the wrong one (doubly wrong of me for not checking this when creating the PR)

I updated it, should me ok now!